charts/r2devops/templates/deployment.yaml

```
@@ -5,12 +5,20 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .name }}
+  annotations:
+    {{- range $key, $val := $.Values.additionalAnnotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
```

Avoid empty `annotation` key if no `additionalAnnotations`:

```suggestion
  {{- with $.Values.additionalAnnotations }}
  annotations:
    {{- range $key, $val := . }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
  {{- end }}
```